cmake/sdkconfig

```diff
@@ -0,0 +1,9 @@
+CONFIG_AUTOSTART_ARDUINO=y
+CONFIG_PARTITION_TABLE_CUSTOM=y
+CONFIG_PARTITION_TABLE_CUSTOM_FILENAME="../DCCppESP32-partitions.csv"
+CONFIG_PARTITION_TABLE_FILENAME="../DCCppESP32-partitions.csv"
```

Comment:
Note new filenames...

Reply:
right - I'll merge into top of the development branch as I should have done before the pull request - sorry.